### ISSUE TITLE
The converters display results are not accessible to the TalkBack screen reader

### ISSUE THREAD

**Issue by k-kolev1985** [nventive/calculator#252]
**Describe the bug**
In the 3 calculator modes, the display (where the digit results are) is read by TalkBack as it should be. In the converter modes however, the displays are not read by TalkBack. When you touch an object or slide your finger on the screen over it, TalkBack should read it, but in the case with the results displays of the converters it does not.

**Steps To Reproduce**
1. Go to Android Settings -> Accessibility -> TalkBack and enable it.
2. Launch "Uno Calculator".
3. In any of the "calculator" modes, slide your finger over the display results. TalkBack reads "Result is", following by a digit value ("0" by default).
4. From the navigation menu, switch to one of the "converter" modes.
5. In the "converter" view, slide your finger over any of the display results areas. TalkBack should read the value for each display, but it does not - it stays silent.

**Expected behavior**
TalkBack should read the value for each display, but it does not - it stays silent.

**Device and Application Information**
 - OS Build: Android 10, February security patch.
 - Architecture: Unknown.
 - Application Version: 1.2.0.
 - Region: Bulgaria
 - Dev Version Installed: Unknown.
